testbenches/project/ad7606x/system_project.tcl

```diff
@@ -15,6 +15,17 @@ source "cfgs/${cfg_file}"
 # Set the project name
 set project_name [file rootname $cfg_file]
 
+# Set project params
+
+global ad_project_params
+
+set INTF $ad_project_params(INTF)
+set ADC_N_BITS $ad_project_params(ADC_N_BITS)
+set NUM_OF_SDI $ad_project_params(NUM_OF_SDI)
```

Comment:
I think that these definitions are not needed here, at least I don't see them used anywhere